### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by allenpg**
added badges, ordered sections according to template, README.md tasks completed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+
+See the [`LICENSE`](/LICENSE) file for details.
+
+## Getting Involved
+
+We would love involvement from more people! If you notice any errors or would like to submit changes, please see our [Contributing Guidelines](./CONTRIBUTING.md). 
+
+We use GitHub issues for [tracking bugs and feature requests](./issues) and Pull Requests (PRs) for [submitting changes](./pulls)
+
+## ...
```
Comment (titaniumbones) [inline README.md:25]: remove extraneous header markup

File: README.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
+## [Optional section(s) on Installation (actually using the service!), Architecture, Dependencies, and Other Considerations]
```
Comment (titaniumbones) [inline README.md:27]: Rename to reflect actual ocntent.

File: README.md [hunk 3]
```diff
@@ -67,8 +90,12 @@ You may need to install Calibre's ebook-convert cli tools. For Mac OS X, these c
 $ ln -s /Applications/calibre.app/Contents/MacOS/ebook-convert /usr/local/bin
 ```
 
-## License
+[fill  out this section if the repo contains deployable/installable code]
```
Comment (titaniumbones) [inline README.md:93]: remove generic template code.

File: README.md [hunk 4]
```diff
@@ -67,8 +90,12 @@ You may need to install Calibre's ebook-convert cli tools. For Mac OS X, these c
...
 
-<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Data Together Learning Materials</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
+## Development
 
-See the [`LICENSE`](/LICENSE) file for details.
+[Step-by-step instructions about how to set up a local dev environment and any dependencies]
+
+## Deployment
```
Comment (titaniumbones) [inline README.md:99]: not relevant for this repo

File: README.md [hunk 5]
```diff
@@ -67,8 +90,12 @@ You may need to install Calibre's ebook-convert cli tools. For Mac OS X, these c
 $ ln -s /Applications/calibre.app/Contents/MacOS/ebook-convert /usr/local/bin
 ```
 
-## License
+[fill  out this section if the repo contains deployable/installable code]
 
-<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Data Together Learning Materials</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
+## Development
```
Comment (titaniumbones) [inline README.md:95]: May not be relevant. If it is, add some text or "Coming soon."

**Comment by dcwalk** [comment]
Other PR handles this, closing for now and will review against #27 to ensure we're covered :)

**Pull request closed without merging**